cli/cli.go

```diff
@@ -45,13 +46,13 @@ func CliArgs(args []string) (*Cliargs, error) {
 	debug := flag.Bool("debug", false, "debug export data")
 	cpuprofile := flag.String("cpuprofile", "", "write cpu profile to `file`")
 	memprofile := flag.String("memprofile", "", "write memory profile to `file`")
-
+	logfile := flag.String("logfile", "", "set loglevel")
```

Comment:
The discription doesn't match the flag and variable name.
And I think we are missing one flag for the log level.